app/controllers/spree/subscriptions_controller.rb

```diff
@@ -23,25 +23,22 @@ def update
     end
 
     def cancel
+      return respond_with_error(t(".stripe_subscription_id_not_found"), 422) unless @subscription.stripe_subscription_id.present?
+    
+      unless @subscription.cancel
```

Comment:
is there a case when `@subscriptoin.cancel` will be false? i guess it always returns true